src/docs/feature-flags/flagpole.mdx

```diff
@@ -0,0 +1,137 @@
+# Flagpole
+Flagpole is our internal, options-backed feature flagging library. 
+
+Features are defined in the [`sentry-options-automator`](https://github.com/getsentry/sentry-options-automator) repository as yaml objects within the [`options/defaults/flagpole.yml`](https://github.com/getsentry/sentry-options-automator/blob/main/options/default/flagpole.yml) file.
+These yaml options are pushed to each of our deployments and stored in a database table, where they are later queried and parsed during feature flag evaluation.
+
+Here's the example configuration for a feature named `organizations:is_sentry`, which is enabled for an organization with the slug `sentry`.
+```yaml
+options:
+  'feature.organizations:is_sentry':
+    created_at: '2024-06-01T00:00:00.000000'
+    enabled: false
+    owner: hybrid-cloud
+    segments:
+    - conditions:
+      - operator: in
+        value:
+          - sentry
+        property: organization_slug
+      name: is_sentry
+      rollout: 100
+```
+
+## Flagpole Feature Yaml Structure
+
+`created_at`
+
+: The ISO datetime of when the feature was added to the config yaml.
```

Comment:
```suggestion
: The ISO 8601 datetime of when the feature was added to the config yaml.
```

or `RFC3339` which should be effectively the same here and has a more accessible spec